src/main/java/racingcar/race/Racing.java

```diff
@@ -1,26 +1,34 @@
 package racingcar.race;
 
-import java.util.ArrayList;
+import racingcar.DiceRacingRule;
+import racingcar.RacingDice;
+
+import java.util.Comparator;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class Racing {
 
-    private final List<Car> carList = new ArrayList<>();
-    private final Dice dice;
+    private static final RacingRule DEFAULT_RACING_RULE = new DiceRacingRule(RacingDice.newInstance());
+
+    private final List<Car> carList;
```

Comment:
인스턴스 변수에서는 자세한 구현체의 이름은 자제해주세요:)
```suggestion
    private final List<Car> cars;
```